packages/toolpad-core/src/AppProvider/AppProvider.tsx

```diff
@@ -1,20 +1,58 @@
 import * as React from 'react';
 import { ThemeProvider } from '@mui/material/styles';
 import CssBaseline from '@mui/material/CssBaseline';
-
 import type { Theme } from '@emotion/react';
+import { baseTheme } from '../themes';
+
+export interface Branding {
+  title?: string;
+  logo?: React.ReactNode;
+}
+
+export interface NavigationPageItem {
+  kind?: 'page';
+  title: string;
+  path?: string;
```

Comment:
With the nomenclature `path` I'd expect it to be the full path. Maybe we need to come up with another name e.g. like "slug"? That way we can later still add `path` if we want to add an override in the future.

```suggestion
  slug: string;
```